apps/site/src/app/(home)/sections/Sponsors/Sponsors.module.scss

```diff
@@ -1,20 +1,20 @@
 @use "zothacks-theme" as theme;
 @use "bootstrap-utils" as utils;
 
-.container {
-	padding-top: 6rem;
-	padding-bottom: 6rem;
-}
+// .container {
+//     padding-top: 6rem;
+//     padding-bottom: 6rem;
+// }
```

Comment:
Chore: little value in keeping around old commented code, better to just remove it